### ISSUE TITLE
Error occurs for Style/MultilineMethodCallIndentation

### ISSUE THREAD

**Issue by NobodysNightmare** [bbatsov/rubocop#2958]
## Expected behavior

Rubocop throws no error messages

## Actual behavior

````
$ rubocop test.rb 
Inspecting 1 file
An error occurred while Style/MultilineMethodCallIndentation cop was inspecting /home/jan/Desktop/test.rb.
To see the complete backtrace run rubocop -d.
.

1 file inspected, no offenses detected

1 error occurred:
An error occurred while Style/MultilineMethodCallIndentation cop was inspecting /home/jan/Desktop/test.rb.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
Mention the following information in the issue report:
0.38.0 (using Parser 2.3.0.6, running on ruby 2.3.0 x86_64-linux)

````

## Steps to reproduce the problem

Given a file called `test.rb`:
````ruby
def foo
  !0
  .nil?
end
````

Run:
````bash
$ rubocop test.rb
````

The problem seems to be with the negation of the first line.

## RuboCop version

```
$ rubocop -V
0.38.0 (using Parser 2.3.0.6, running on ruby 2.3.0 x86_64-linux)
```

**Comment by alexdowad** [comment]
Just added a fix to my open PR.
